soc/sensry/ganymed/sy1xx/Kconfig

```diff
@@ -0,0 +1,147 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 sensry.io
+
+config SOC_SERIES_SY1XX
+    bool
+	select RISCV
+    select RISCV_ISA_RV32I
+    select RISCV_ISA_EXT_M
+    select RISCV_ISA_EXT_ZICSR
+    select RISCV_PRIVILEGED
+
+
+if SOC_SERIES_SY1XX
+
+config RISCV_PMP
+    default n
+
+config RISCV_HAS_CLIC
+    default n
+
+config RISCV_VECTORED_MODE
+    default y
+
+config INCLUDE_RESET_VECTOR
+    default y
+
+config GEN_IRQ_VECTOR_TABLE
+    default y
+
+config RISCV_GENERIC_TOOLCHAIN
+	default y if "$(ZEPHYR_TOOLCHAIN_VARIANT)" = "zephyr"
+
+config RV_BOOT_HART
+    # default cluster id 0x3e, core 0 (FC) => 0x3e0 == 992
+    default 992
+
+config RISCV_SOC_CONTEXT_SAVE
+	default n
+
+config RISCV_SOC_OFFSETS
+	default n
+
+config NUM_IRQS
+    default 32
+
+config RISCV_SOC_INTERRUPT_INIT
+	default y
+
+config RISCV_SOC_HAS_ISR_STACKING
+    default n
+
+config DYNAMIC_INTERRUPTS
+    default y
+
+config GEN_ISR_TABLES
+    default y
+
+config RISCV_MCAUSE_EXCEPTION_MASK
+	default 0x1F
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 1000
+
+config SYS_CLOCK_TICKS_PER_SEC
+    default 1000
+
+config BOOTLOADER_MCUBOOT
+    default n
+
+config USE_DT_CODE_PARTITION
+    default y
+
+config RISCV_SOC_HAS_CUSTOM_IRQ_LOCK_OPS
+    default n
+
+config RISCV_SOC_EXCEPTION_FROM_IRQ
+    default n
+
+config INIT_STACKS
+    default y
+
+## scheduling
+config TIMESLICING
+    default y
+
+config TIMESLICE_PER_THREAD
+    default y
+
+config TIMESLICE_SIZE
+    default 10
+
+
+## general
+config BOOT_BANNER
+    default y
+
+config XIP
+    default n
+
+## serial
+config SERIAL_SUPPORT_INTERRUPT
+    default n
+
+config UART_ASYNC_API
+    default n
+
+config SERIAL_HAS_DRIVER
+    default y
+
+config SERIAL
+    default y
+
+## console
+config CONSOLE
+    default y
+
+config UART_CONSOLE
+    default y
+
+config CONSOLE_HANDLER
+    default y
+
+config PRINTK
+    default y
+
+config STDOUT_CONSOLE
+    default y
+
+config EARLY_CONSOLE
+    default y
+
+
+## shell
+config SHELL
+    default y
+config KERNEL_SHELL
+    default y
+config SHELL_BACKEND_SERIAL
```

Comment:
No to any of this

Reply:
moved this to the board, should not be in soc